app/controllers/hiring_staff/vacancies_controller.rb

```diff
@@ -32,7 +32,7 @@ def review
     unless @vacancy.valid?
       redirect_to_incomplete_step
     else
-      state = params[:edit_draft] == 'true' ? 'edit' : 'review'
+      state = params[:edit_draft] == 'true' ? 'edit' :  @vacancy&.state == 'copy' ? 'copy' : 'review'
```

Comment:
It's not a good practice to have a ternary inside a ternary. Maybe you can move that to a method